.github/workflows/macstadium-ios-e2e.yml

```diff
@@ -158,22 +157,32 @@ jobs:
         run: |
           rm -rf /Users/administrator/.cocoapods/repos/cocoapods/.git/index.lock
           yarn install-bundle && yarn install-pods
-      
+
       - uses: irgaly/xcode-cache@v1
         with:
           key: xcode-cache-deriveddata-${{ github.workflow }}-${{ github.sha }}
           restore-keys: xcode-cache-deriveddata-${{ github.workflow }}-
-      
-      - name: Modify env and build app in release mode
-        run: |
-          sed -i'' -e "s/IS_TESTING=false/IS_TESTING=true/" .env && rm -f .env-e 
-          yarn detox build --configuration ios.sim.release
-      - name: Detox iOS e2e tests serial
-        run: |
-          ./scripts/run-serial-e2e.sh 3
 
-      - name: Detox iOS e2e tests parallel
+      - name: Install Maestro
+        run: curl -fsSL "https://get.maestro.mobile.dev" | bash
```

Comment:
Maestro releases seem to be put out quite often, can we select a specific version instead (same android flow)